python/src/tutorials/tutorial1.ipynb

```diff
@@ -0,0 +1,285 @@
+      "\u001b[0;31mTypeError\u001b[0m: ChannelHyperparameters() takes no arguments"
```

Comment:
nit: it seems like there was a type error here for this run. was the notebook result updated? 

<img width="1075" alt="image" src="https://github.com/user-attachments/assets/1ea9ef00-a250-442e-acef-ace324d63273">

Reply:
Fixed this.